libsolidity/lsp/FileRepository.h

```diff
@@ -29,7 +29,10 @@ class FileRepository
 {
 public:
 	explicit FileRepository(boost::filesystem::path const& _basePath):
-		m_fileReader(_basePath) {}
+		m_fileReader(_basePath)
+	{
+		m_fileReader.allowDirectory(boost::filesystem::path("/"));
```

Comment:
Does this work on windows?

Reply:
Yes, with my changes added to this PR: All tests pass (locally).